itou/users/models.py

```diff
@@ -103,6 +103,10 @@ def save(self, *args, **kwargs):
 
         super().save(*args, **kwargs)
 
+    def clean(self, *args, **kwargs):
+        if self.email and User.email_already_exists(self.email, exclude_pk=self.pk):
+            raise ValidationError({"email": self.ERROR_EMAIL_ALREADY_EXISTS})
+
```

Comment:
Euh du coup si j'ai bien suivi tu peux virer https://github.com/betagouv/itou/blob/master/itou/users/models.py#L94-L99 qui devient redondant non?

Reply:
ERRATUM `a model’s clean() method is not invoked when you call your model’s save() method.` vu sur https://docs.djangoproject.com/en/3.1/ref/models/instances/#django.db.models.Model.clean

Du coup peut-être factoriser les deux bouts de code identiques?

Reply:
Yes on peut factoriser, c'est fait